rules/camunda-cloud/no-loop.js

```diff
@@ -122,6 +122,8 @@ function getNextFlowElements(flowElement) {
         }
       }
     }
+
+    return [];
```

Comment:
From what I understand we either dive into the called process (when invoked via a call activity) _or_ fallback to the default behavior, which is to [continue the normal flow](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/156/files#diff-8be5b0397c4f3421ba20b4356cc47b4d0f0629542fadbab5afa4c6434d9d3289R138). Is my assumption correct?

Then why would we want to not continue the normal flow if referenced start event cannot be found / called reference cannot resolve?

Reply:
It seems I got a little bit confused here. There is no bug and the behavior was correct. 🙈 I will close this pull request and the issue and leave a message in the Slack thread.

Reply:
:steam_locomotive: